docs/tutorials.rst

```diff
@@ -32,70 +33,100 @@ save it as ``rockcraft.yaml``:
-  Executed: build hello                                                                                                       
-  Executed: stage hello                                                                                                       
-  Executed: prime hello                                                                                                       
-  Executed parts lifecycle                                                                                                    
-  Created new layer                                                                                                           
-  Cmd set to ['/usr/bin/hello', '-t']                                                                                         
-  Labels and annotations set to ['org.opencontainers.image.version=1.0', 'org.opencontainers.image.title=hello', 'org.opencontainers.image.ref.name=hello', 'org.opencontainers.image.licenses=Apache-2.0', 'org.opencontainers.image.created=2022-06-30T09:07:38.124741+00:00']                                                                                                        
-  Exported to OCI archive 'hello_1.0_amd64.rock'  
+    Launching instance...
+    Retrieved base ubuntu:20.04
+    Extracted ubuntu:20.04
+    Executed: pull hello
+    Executed: overlay hello
+    Executed: build hello
+    Executed: stage hello
+    Executed: prime hello
+    Executed parts lifecycle
+    Created new layer
+    Cmd set to ['/usr/bin/hello', '-t']
+    Labels and annotations set to ['org.opencontainers.image.version=1.0', 'org.opencontainers.image.title=hello', 'org.opencontainers.image.ref.name=hello', 'org.opencontainers.image.licenses=Apache-2.0', 'org.opencontainers.image.created=2022-06-30T09:07:38.124741+00:00']
+    Exported to OCI archive 'hello_1.0_amd64.rock'
 
 At the end of the process, a file named ``hello_1.0_amd64.rock`` should be
-present in the current directory.
+present in the current directory. That's your ROCK, in oci-archive format.
 
-Running OCI image in docker
----------------------------
 
-First, import the recently created OCI Image into docker:
+Run the ROCK in Docker
+----------------------
+
+First, import the recently created ROCK into Docker:
 
 .. code-block:: sh
 
-  skopeo --insecure-policy copy oci-archive:hello_1.0_amd64.rock docker-daemon:hello:1.0
+    skopeo --insecure-policy copy oci-archive:hello_1.0_amd64.rock docker-daemon:hello:1.0
 
-Now run the ``hello`` command from the OCI image:
+Now run the ``hello`` command from the ROCK:
 
 .. code-block:: sh
 
-  $ docker run hello:1.0
+    $ docker run hello:1.0
 
 Which should print:
 
 .. code-block:: sh
 
-  hello, world
+    hello, world
+
+
+Publish a ROCK to a registry
+============================
+
+Prerequisites
+-------------
+
+- skopeo installed https://github.com/containers/skopeo
```

Comment:
docker has parent, skopeo doesn't

Reply:
what do you mean by "parent"?

Reply:
(I think you meant parenthesis :p if so, that's fixed now)